ros2controlcli/ros2controlcli/api/__init__.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2019 Canonical Ldt.
```

Comment:
Is copyright correct? Is this code ported from the ROS1?

Reply:
I think that should be changed, year and copyright holder.

Reply:
Fixed.